test/forwarddiff.jl

```diff
@@ -130,3 +130,31 @@ end
     fd2 = FiniteDifferences.central_fdm(5, 1)(erfcα, x0)
     @test norm(fd1 - fd2) < tol
 end
+
+@testset "LocalNonlinearity sensitivity using ForwardDiff" begin
+    function compute_force(ε::T) where {T}
+        # solve the 1D Gross-Pitaevskii equation with ElementGaussian potential
+        a = 10.0
+        lattice = a .* [[1 0 0.]; [0 0 0]; [0 0 0]]
+        positions = [[0.2, 0, 0], [0.8, 0, 0]]
+        gauss = ElementGaussian(1.0, 0.5)
+        atoms = [gauss, gauss]
+        C = 1.0
+        α = 2
```

Comment:
It's a test ... just hard code these values.

Reply:
[6a679f5](https://github.com/JuliaMolSim/DFTK.jl/pull/779/commits/6a679f54db4441f1e6c26e88546555fca7368e7d)